### PR TITLE
Nwd nnw

### PR DESCRIPTION
**Pull request by Martyna-Tu**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @Martyna-Tu

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Martyna-Tu for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,22 @@
 #pragma once
+#include <cstdlib>
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    int temp;
+    while(rhs!=0){
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: Add space between operations like '+' '-' '!=' '==' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+	    temp=rhs;
+	    rhs=lhs%rhs;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:8]: same here

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,22 @@
 #pragma once
+#include <cstdlib>
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    int temp;
+    while(rhs!=0){
+	    temp=rhs;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:7]: same here

File: homework/nwd-nnw/nwdNww.hpp [hunk 4]
```diff
@@ -1,11 +1,22 @@
...
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+	
+    int nwd = NWD(lhs, rhs);
+    if(nwd == 0){
+	return 0;
+    }else {
+	 return abs(lhs*rhs/nwd);
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:20]: If you return variable here, you don't need to use `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**